packages/nextra-theme-blog/src/mdx-theme.tsx

```diff
@@ -45,12 +44,18 @@ const HeaderLink = ({
   const slug = slugger.slug(ReactDOMServer.renderToStaticMarkup(children))
   return (
     <Tag {...props}>
-      <span className="subheading-anchor" id={slug} />
```

Comment:
I got the same error as in https://github.com/shuding/nextra/pull/481
This element is redundant, `<Tag/>` already receives `id` from props

Reply:
Can we remove the `id` in `<Tag/>` but keep this anchor element? https://github.com/shuding/nextra/pull/481#issuecomment-1192706522

Reply:
@shuding done, now it works as expected, I took the same `marginTop` that was specified before

before:
![image](https://user-images.githubusercontent.com/7361780/180485757-e47b10ea-7788-43a0-9766-4dae5b1420cd.png)
after:
![image](https://user-images.githubusercontent.com/7361780/180485787-7baf700e-4c7e-4a57-aa58-6763738c7fbe.png)